[2025-01-30T09:33:15.998+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-30T09:33:16.055+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: homework_workflow_ingest.download_data scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2025-01-30T09:33:16.070+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: homework_workflow_ingest.download_data scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2025-01-30T09:33:16.071+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-30T09:33:16.116+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): download_data> on 2019-03-02 00:00:00+00:00
[2025-01-30T09:33:16.134+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=197) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-30T09:33:16.135+0000] {standard_task_runner.py:72} INFO - Started process 205 to run task
[2025-01-30T09:33:16.133+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'homework_workflow_ingest', 'download_data', 'scheduled__2019-03-02T00:00:00+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/homework_dag.py', '--cfg-path', '/tmp/tmpu78ygd4w']
[2025-01-30T09:33:16.136+0000] {standard_task_runner.py:105} INFO - Job 38: Subtask download_data
[2025-01-30T09:33:16.284+0000] {task_command.py:467} INFO - Running <TaskInstance: homework_workflow_ingest.download_data scheduled__2019-03-02T00:00:00+00:00 [running]> on host 710324c6e27a
[2025-01-30T09:33:16.566+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='homework_workflow_ingest' AIRFLOW_CTX_TASK_ID='download_data' AIRFLOW_CTX_EXECUTION_DATE='2019-03-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-03-02T00:00:00+00:00'
[2025-01-30T09:33:16.567+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-30T09:33:16.607+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-30T09:33:16.608+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'curl -o /opt/***/data/yellow_2019-03_data.parquet https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2019-03.parquet']
[2025-01-30T09:33:16.650+0000] {subprocess.py:99} INFO - Output:
[2025-01-30T09:33:16.781+0000] {subprocess.py:106} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2025-01-30T09:33:16.781+0000] {subprocess.py:106} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2025-01-30T09:33:24.553+0000] {subprocess.py:106} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0  110M    0  189k    0     0   173k      0  0:10:51  0:00:01  0:10:50  173k  9  110M    9 11.0M    0     0  5274k      0  0:00:21  0:00:02  0:00:19 5274k 28  110M   28 31.2M    0     0   9.9M      0  0:00:11  0:00:03  0:00:08  9.9M 43  110M   43 48.0M    0     0  11.7M      0  0:00:09  0:00:04  0:00:05 11.7M 58  110M   58 64.4M    0     0  12.6M      0  0:00:08  0:00:05  0:00:03 13.2M 75  110M   75 83.2M    0     0  13.4M      0  0:00:08  0:00:06  0:00:02 16.2M 93  110M   93  103M    0     0  14.4M      0  0:00:07  0:00:07 --:--:-- 18.4M100  110M  100  110M    0     0  14.2M      0  0:00:07  0:00:07 --:--:-- 17.1M
[2025-01-30T09:33:24.557+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-30T09:33:26.797+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-30T09:33:26.798+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=homework_workflow_ingest, task_id=download_data, run_id=scheduled__2019-03-02T00:00:00+00:00, execution_date=20190302T000000, start_date=20250130T093316, end_date=20250130T093326
[2025-01-30T09:33:30.630+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-30T09:33:30.675+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-30T09:33:31.169+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-01-30T09:33:31.762+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-01-30T09:47:55.129+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-30T09:47:55.161+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: homework_workflow_ingest.download_data scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2025-01-30T09:47:55.176+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: homework_workflow_ingest.download_data scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2025-01-30T09:47:55.177+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-30T09:47:55.200+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): download_data> on 2019-03-02 00:00:00+00:00
[2025-01-30T09:47:55.221+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=853) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-30T09:47:55.223+0000] {standard_task_runner.py:72} INFO - Started process 860 to run task
[2025-01-30T09:47:55.222+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'homework_workflow_ingest', 'download_data', 'scheduled__2019-03-02T00:00:00+00:00', '--job-id', '86', '--raw', '--subdir', 'DAGS_FOLDER/homework_dag.py', '--cfg-path', '/tmp/tmpbt5n7wo7']
[2025-01-30T09:47:55.224+0000] {standard_task_runner.py:105} INFO - Job 86: Subtask download_data
[2025-01-30T09:47:55.319+0000] {task_command.py:467} INFO - Running <TaskInstance: homework_workflow_ingest.download_data scheduled__2019-03-02T00:00:00+00:00 [running]> on host 710324c6e27a
[2025-01-30T09:47:55.503+0000] {abstractoperator.py:783} ERROR - Exception rendering Jinja template for task 'download_data', field 'bash_command'. Template: 'curl -o {{ var.value.AIRFLOW_HOME }}/data/{{ params.trip_class }}_{{ ds[:7] }}_data.parquet https://d37ci6vzurychx.cloudfront.net/trip-data/{{ params.trip_class }}_tripdata_{{ ds[:7] }}.parquet'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/abstractoperator.py", line 775, in _do_render_template_fields
    rendered_content = self.render_template(
                       ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 171, in render_template
    return self._render(template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/abstractoperator.py", line 730, in _render
    return super()._render(template, context, dag=dag)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 127, in _render
    return render_template_to_string(template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 301, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 296, in render_template
    return "".join(nodes)
           ^^^^^^^^^^^^^^
  File "<template>", line 15, in root
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/sandbox.py", line 327, in getattr
    value = getattr(obj, attribute)
            ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/context.py", line 126, in __getattr__
    self.var = Variable.get(key, deserialize_json=self._deserialize_json)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/variable.py", line 145, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable AIRFLOW_HOME does not exist'
[2025-01-30T09:47:55.514+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3115, in _execute_task_with_callbacks
    task_orig = self.render_templates(context=context, jinja_env=jinja_env)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3534, in render_templates
    original_task.render_template_fields(context, jinja_env)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 1438, in render_template_fields
    self._do_render_template_fields(self, self.template_fields, context, jinja_env, set())
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/abstractoperator.py", line 775, in _do_render_template_fields
    rendered_content = self.render_template(
                       ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 171, in render_template
    return self._render(template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/abstractoperator.py", line 730, in _render
    return super()._render(template, context, dag=dag)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 127, in _render
    return render_template_to_string(template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 301, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 296, in render_template
    return "".join(nodes)
           ^^^^^^^^^^^^^^
  File "<template>", line 15, in root
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/sandbox.py", line 327, in getattr
    value = getattr(obj, attribute)
            ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/context.py", line 126, in __getattr__
    self.var = Variable.get(key, deserialize_json=self._deserialize_json)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/variable.py", line 145, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable AIRFLOW_HOME does not exist'
[2025-01-30T09:47:55.588+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=homework_workflow_ingest, task_id=download_data, run_id=scheduled__2019-03-02T00:00:00+00:00, execution_date=20190302T000000, start_date=20250130T094755, end_date=20250130T094755
[2025-01-30T09:47:55.617+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-30T09:47:55.623+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 86 for task download_data ('Variable AIRFLOW_HOME does not exist'; 860)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3115, in _execute_task_with_callbacks
    task_orig = self.render_templates(context=context, jinja_env=jinja_env)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3534, in render_templates
    original_task.render_template_fields(context, jinja_env)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 1438, in render_template_fields
    self._do_render_template_fields(self, self.template_fields, context, jinja_env, set())
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/abstractoperator.py", line 775, in _do_render_template_fields
    rendered_content = self.render_template(
                       ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 171, in render_template
    return self._render(template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/abstractoperator.py", line 730, in _render
    return super()._render(template, context, dag=dag)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 127, in _render
    return render_template_to_string(template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 301, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 296, in render_template
    return "".join(nodes)
           ^^^^^^^^^^^^^^
  File "<template>", line 15, in root
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/sandbox.py", line 327, in getattr
    value = getattr(obj, attribute)
            ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/context.py", line 126, in __getattr__
    self.var = Variable.get(key, deserialize_json=self._deserialize_json)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/variable.py", line 145, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable AIRFLOW_HOME does not exist'
[2025-01-30T09:47:55.687+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-01-30T09:47:55.734+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-30T09:47:55.755+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-01-30T09:58:37.073+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-30T09:58:37.126+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: homework_workflow_ingest.download_data scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2025-01-30T09:58:37.149+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: homework_workflow_ingest.download_data scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2025-01-30T09:58:37.150+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-30T09:58:37.261+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): download_data> on 2019-03-02 00:00:00+00:00
[2025-01-30T09:58:37.274+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'homework_workflow_ingest', 'download_data', 'scheduled__2019-03-02T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/homework_dag.py', '--cfg-path', '/tmp/tmp5shirpiy']
[2025-01-30T09:58:37.276+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask download_data
[2025-01-30T09:58:37.277+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=197) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-30T09:58:37.278+0000] {standard_task_runner.py:72} INFO - Started process 204 to run task
[2025-01-30T09:58:37.402+0000] {task_command.py:467} INFO - Running <TaskInstance: homework_workflow_ingest.download_data scheduled__2019-03-02T00:00:00+00:00 [running]> on host 713eb9fcfb63
[2025-01-30T09:58:37.669+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='homework_workflow_ingest' AIRFLOW_CTX_TASK_ID='download_data' AIRFLOW_CTX_EXECUTION_DATE='2019-03-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-03-02T00:00:00+00:00'
[2025-01-30T09:58:37.670+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-30T09:58:37.692+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-30T09:58:37.693+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'curl -o /opt/***/data/yellow_2019-03_data.parquet https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2019-03.parquet']
[2025-01-30T09:58:37.714+0000] {subprocess.py:99} INFO - Output:
[2025-01-30T09:58:37.743+0000] {subprocess.py:106} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2025-01-30T09:58:37.745+0000] {subprocess.py:106} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2025-01-30T09:58:44.656+0000] {subprocess.py:106} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  5  110M    5 6154k    0     0  5405k      0  0:00:20  0:00:01  0:00:19 5403k 24  110M   24 27.2M    0     0  12.6M      0  0:00:08  0:00:02  0:00:06 12.5M 45  110M   45 50.7M    0     0  15.7M      0  0:00:07  0:00:03  0:00:04 15.7M 63  110M   63 69.9M    0     0  16.9M      0  0:00:06  0:00:04  0:00:02 16.9M 74  110M   74 82.7M    0     0  15.1M      0  0:00:07  0:00:05  0:00:02 15.8M 86  110M   86 95.9M    0     0  15.3M      0  0:00:07  0:00:06  0:00:01 17.6M100  110M  100  110M    0     0  15.9M      0  0:00:06  0:00:06 --:--:-- 17.5M
[2025-01-30T09:58:44.683+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-30T09:58:45.317+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-30T09:58:45.364+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=homework_workflow_ingest, task_id=download_data, run_id=scheduled__2019-03-02T00:00:00+00:00, execution_date=20190302T000000, start_date=20250130T095837, end_date=20250130T095845
[2025-01-30T09:58:46.711+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-30T09:58:46.741+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-01-30T09:58:47.285+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
